common/changes/cadl-vs/compliancefix_2023-01-31-08-20.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "cadl-vs",
+      "comment": "Explicitly adding NewtonSoft.Json 13.0.2 to address vulnerability",
```

Comment:
```suggestion
      "comment": "Updating NewtonSoft.Json to 13.0.2",
```

Nit: This doesn't actually close any vulnerability in this case. If VS has v13.0.2 we will use it without this change, and if it doesn't we won't use it even with this change.